### ISSUE TITLE
Create transcripts for our training videos

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4110]
## Area or Component
Training videos on YouTube
Example: https://www.youtube.com/watch?v=qVkeizogmeQ&feature=emb_title

These are linked on the static site.

## Quick Summary
We don't have video transcripts for our training videos.

## Accessibility Violation
Multimedia should include a transcript: https://dequeuniversity.com/class/multimedia/transcripts/when-needed

## Action Requested
Add transcripts for our training videos.

**Comment by lauraykerr** [comment]
[Youtube caption/transcript from Jamie](https://docs.google.com/document/d/1sumJj5fuVMr7eHOc5E-CmqYkKx_VC_BUE5yRMvasXGY/edit?pli=1#heading=h.9ccwiab8962h)

**Comment by lauraykerr** [comment]
**Some background to recommended solution:**
- Right now, the SimpleReport training videos on Youtube do have automated transcripts, which you can access from the three dots at the end of the menu below the video titles ("show transcript").
- Ideally, to improve accessibility, we would create these manually, which video owners have the option of doing. 
- USDS owns these videos. 
- There's some talk on our team of updating these videos with the release of CSV uploader, additional diseases, etc. 
- For the [one video embedded on the static site](https://www.simplereport.gov/support/), there's no option to show transcript if playing from the site. 

**Recommended solution:** 
- Create a transcript for the video embedded on the Support page and link to it under the video (would need some design work). 
- Host transcript on static site in simple page, like [Terms of service](https://www.simplereport.gov/terms-of-service/).
- Revisit Youtube transcripts if/when we redo videos.

**Comment by lauraykerr** [comment]
![SR video transcript.png](https://images.zenhubusercontent.com/62619ba9c523e0cc6c6fbf1e/224d5db9-bbd5-42ff-a0a8-862920327f90)
Transcript link, per conversation with @kenieh:
-  16px p text
-  12px space between text and video
-  Left-aligned 
-  16px icon (attached) ![up-right-from-square-light.svg](https://images.zenhubusercontent.com/62619ba9c523e0cc6c6fbf1e/9f67460c-42b4-4841-a03e-0e782c3c1236)

**Comment by emmastephenson** [comment]
Perfect, thanks @lauraykerr and @kenieh! Do you want to pair with an engineer to get the link placed on the static site?

**Comment by kenieh** [comment]
@lauraykerr YESSS this looks great! @emmastephenson happy to pair1

**Comment by lauraykerr** [comment]
@emmastephenson That would be great! I think I can first create the actual transcript page on my own, then reach out pair with an engineer to get the link on the Support page

**Comment by lauraykerr** [comment]
PR #[365](https://github.com/CDCgov/prime-simplereport-site/pull/365) adds a transcript page and link below the support page video embed
